src/components/addComments.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <section class="add_message_section">
+    <form @submit.prevent="addMessageHandler" class="message_form">
+      <textarea
+        class="message_textarea"
+        v-model="currentTextAreaValue"
+        name="message"
+        id="message"
+        cols="30"
+        rows="10"
+        placeholder="Add a comment…"
+        required
+      />
+      <img
+        class="avatar"
+        width="32"
+        height="32"
+        :src="require(`../photos/photosForProfile/${currentUser.pathToPhoto}`)"
+        alt="avatar"
+      />
+      <button class="send_message" type="submit">SEND</button>
+    </form>
+
+    <select
+      @change="
+        (event) => {
+          changeCurrentUser(Number(event.target.value));
+        }
+      "
```

Comment:
in Vue it's common not to use arrow functions in event handlers. to get access to `event` Vue provides you with a special `$event` variable.
```suggestion
      @change="changeCurrentUser(Number($event.target.value))"
```